app/artifact-cas/internal/conf/conf.proto

```diff
@@ -66,6 +66,7 @@ message Credentials {
     oneof backend {
         AWSSecretManager aws_secret_manager = 1;
```

Comment:
It would be great to also update references in the readme for the supported backends and add the new one :)

i.e https://github.com/chainloop-dev/chainloop/blob/main/app/artifact-cas/README.md#system-dependencies